python/ray/train/_internal/checkpoint.py

```diff
@@ -96,15 +96,11 @@ def _load_checkpoint(
             # Load checkpoint from path.
             return load_checkpoint_from_path(checkpoint_to_load)
 
-    def _process_checkpoint(
+    def _process_one_checkpoint(
```

Comment:
Nit: Keep this one as `_process_checkpoint` and name the next one `_process_checkpoints`.

Reply:
sure, done